x-pack/legacy/plugins/ml/server/models/job_validation/messages.js

```diff
@@ -271,14 +286,28 @@ export const getMessages = () => {
       }),
       url: 'https://www.elastic.co/guide/en/elasticsearch/reference/{{version}}/ml-job-resource.html#ml-job-resource'
     },
+    job_group_id_invalid_max_length: {
+      status: 'ERROR',
+      text: i18n.translate('xpack.ml.models.jobValidation.messages.jobGroupIdInvalidMaxLengthErrorMessage', {
+        defaultMessage: 'Job group name must be more than {maxLength, plural, one {# character} other {# characters}} long.',
```

Comment:
Typo in here. Should be `Job group name must not be more`.... (missing the word 'not'!)